test/unittest/component/test_serialization.py

```diff
@@ -27,15 +27,19 @@ def test_model():
 
 
 @pytest.mark.skipif(not torch, reason='Torch not installed')
-def test_sklearn(empty):
+def test_sklearn(local_empty_data_layer):
     m = Estimator(
         identifier='test',
         object=SVC(),
         encoder=tensor(torch.float, shape=(32,)),
     )
-    empty.add(m)
-    assert empty.metadata.component_collection.count_documents({}) == 2
-    pprint.pprint(empty.metadata.get_component(type_id='model', identifier='test'))
-    reloaded = empty.load(type_id='model', identifier='test')
+    local_empty_data_layer.add(m)
+    assert local_empty_data_layer.metadata.component_collection.count_documents({}) == 2
+    pprint.pprint(
+        local_empty_data_layer.metadata.get_component(
+            type_id='model', identifier='test'
+        )
+    )
+    reloaded = local_empty_data_layer.load(type_id='model', identifier='test')
     assert isinstance(reloaded.object, Artifact)
     pprint.pprint(reloaded)
```

Comment:
print?

Reply:
I removed all the print functions used in the unittests and changed them to asserts.